### ISSUE TITLE
Download from a single user

### ISSUE THREAD

**Issue by panhartstuff** [Seklfreak/discord-image-downloader-go#136]
Would be nice to have the ability to limit downloads to a single user

**Comment by matslarson** [comment]
Agreed, filtering specific users either in or out (whitelist/blacklist) would be a very useful addition

**Comment by drachedachte1** [comment]
you could like make a string into the program (this is just a idea i didnt look at the code dont know what programming language it uses and rn i dont rly care)
if {pictures count == 10}
  break (or something like that
else
continue
